### PR TITLE
run sib with kinect

### PR DESCRIPTION
**Pull request by yuyu2172**
When running sib with kinect, run `sib_kinect.lauch` instead of `setup_softkinetic.launch`.

### CODE REVIEW

**Comment by knorth55** [comment]
conflict

**Comment by yuyu2172** [comment]
fixed

File: jsk_2016_01_baxter_apc/launch/sib_kinect.launch [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        pkg="jsk_topic_tools" type="standalone_complexed_nodelet"
+        respawn="true" output="screen">
+  </node>
+
+  <include file="$(find jsk_2016_01_baxter_apc)/launch/segmentation_in_bin.launch">
+    <arg name="NODELET_MANAGER_RIGHT" value="$(arg NODELET_MANAGER_RIGHT)" />
+    <arg name="NODELET_MANAGER_LEFT" value="$(arg NODELET_MANAGER_LEFT)" />
+    <arg name="NODELET_ID_0" value="$(arg NODELET_ID_0)" />
+    <arg name="NODELET_ID_1" value="$(arg NODELET_ID_1)" />
+    <arg name="RIGHT_INPUT_IMAGE" default="kinect2_torso/hd/image_color_rect" />
```
Comment (knorth55) [inline jsk_2016_01_baxter_apc/launch/sib_kinect.launch]: default -> value

Comment (yuyu2172) [inline jsk_2016_01_baxter_apc/launch/sib_kinect.launch]: fixed

**Comment by knorth55** [comment]
+1

**Pull request merged**